processor/src/test/java/org/mapstruct/ap/test/template/InheritConfigurationTest.java

```diff
@@ -53,7 +53,7 @@ public void shouldInheritConfigurationSingleCandidates() {
         assertThat( createdTarget ).isNotNull();
         assertThat( createdTarget.getStringPropY() ).isEqualTo( "1" );
         assertThat( createdTarget.getIntegerPropY() ).isEqualTo( 2 );
-        assertThat( createdTarget.getNestedResultProp() ).isEqualTo( "nested");
+        assertThat( createdTarget.getNestedResultProp() ).isEqualTo( "nested" );
```

Comment:
Funny story. There's a whole bunch of stuff that the checkstyle version we currently use misses. I found that with the updated version that is used in the Eclipse checkstyle plugin (for the same rule set). I'll send a PR for updating checkstyle in the build and for fixing the thereby found issues as well. I have it prepared locally already.